src/collections/_documentation/development/sdk-dev/event-payloads/index.md

```diff
@@ -3,6 +3,36 @@ title: Event Payloads
 sidebar_order: 3
 ---
 
+Events are the fundamental data that clients, often through use of an SDK, send
+to the Sentry server.
```

Comment:
I realized we never formally define "event" in the documentation.
Started with something simple.

In `src/collections/_documentation/workflow/search.md` we say:

> Events are the underlying event data captured using Sentry SDKs (read: errors and exceptions).

Though that is not precise because there is `capture_message` and perf metrics in spans that are not errors/exceptions.